src/Core/Package.vala

```diff
@@ -701,11 +713,11 @@ public class AppCenterCore.Package : Object {
                     break;
 
                 case AppStream.IconKind.UNKNOWN:
-                    warning ("'%s' is an unknown kind of AppStream icon", _icon.get_name ());
+                    //warning ("'%s' is an unknown kind of AppStream icon", _icon.get_name ());
```

Comment:
This change and the one below is not essential for the purpose of this PR is it?